lib/matplotlib/backends/_backend_tk.py

```diff
@@ -717,7 +754,8 @@ def __init__(self, toolmanager, window):
     def add_toolitem(
             self, name, group, position, image_file, description, toggle):
         frame = self._get_groupframe(group)
-        button = self._Button(name, image_file, toggle, frame)
+        button = NavigationToolbar2Tk._Button(self, name, image_file, toggle,
```

Comment:
Might as well make this an un-bound top level function?

Reply:
I've been redoing most of the MEP22 implementation to redirect to NavigationToolbar2 everywhere instead of having duplicate implementations; I'd rather keep things there and if we ever switch to MEP22-by-default (which I would rather not, I have deep misgivings about that API design discussed at length elsewhere, but no need to relitigate that here), move everything to the MEP22 classes and point NavigationToolbar2 to them.  IOW: I don't think it's worth turning this into a free function "just because" MEP22 also uses it.